frontend/src/Components/UpdateBook.tsx

```diff
@@ -0,0 +1,50 @@
+import {Book} from "../Model/Book";
+import React, {useState} from "react";
+
+type UpdateBookProps = {
+    book: Book;
+    updateBook: (updateBook: Book) => void;
+}
+
+export default function UpdateBook(props: UpdateBookProps) {
+    const [title, setTitle] = useState(props.book.title);
+    const [author, setAuthor] = useState(props.book.author);
+    const [description, setDescription] = useState(props.book.description);
+    const [imageUrl, setImageUrl] = useState(props.book.imageUrl);
+    const [isbn, setIsbn] = useState(props.book.isbn);
+    const [pages, setPages] = useState(props.book.pages);
+    const [year, setYear] = useState(props.book.year);
+    const [genre, setGenre] = useState(props.book.genre);
```

Comment:
Hast du einen bestimmten Grund, warum du hier und in der AddNewBook component für jede Property einen einzelnen State gemacht hast, statt einen State für das ganze Buch?
```
const [bookForm, setBookForm] = useState<Book>(props.book);
```

Reply:
Nach Rücksprache: Wird für Refactoring notiert